### PR TITLE
airmap: Update dependency list

### PR DESCRIPTION
**Pull request by anselmolsm**
@broody

### CODE REVIEW

File: recipes-airmap/airmap/AirMapSDK-Embedded_1.5.0.bb [hunk 1]
```diff
@@ -8,12 +8,12 @@ DESCRIPTION = "The AirMap airspace management system facilitates \
 LICENSE = "Apache-2.0"
 LIC_FILES_CHKSUM="file://License.txt;md5=d515a6d920f942dc31ac8432e8ce4b71"
 
-RDEPENDS_${PN} += "python-requests python-json libgps libgpsd gpsd-conf gpsd-gpsctl gps-utils python-paho-mqtt"
+RDEPENDS_${PN} += "python-requests python-json libgps libgpsd gpsd-conf gpsd-gpsctl gps-utils python-paho-mqtt python-cryptography protobuf ntp python-pyyaml python-dev"
```
Comment (anselmolsm) [inline recipes-airmap/airmap/AirMapSDK-Embedded_1.5.0.bb]: Do we really need python-dev?

Comment (broody) [inline recipes-airmap/airmap/AirMapSDK-Embedded_1.5.0.bb]: Hi Anselmo, yes it looks like the AirmapSDK needs python-dev.

Comment (lucasdemarchi) [inline recipes-airmap/airmap/AirMapSDK-Embedded_1.5.0.bb]: @anselmolsm @broody this is a huge list of dependencies. Taking a quick look on their source it's not obvious why python-dev or the others are needed. We will need more information on this and would be good to get @airmap or @airmapinc to comment here.

**Comment by lucasdemarchi** [comment]
Closing this for now. More info is needed to consider if this is correct.

**Pull request closed without merging**